src/modules/header/header.jsx

```diff
@@ -121,8 +121,10 @@ const Header = ({ children, className, id, ...passProps }) => {
       tag="header"
       {...passProps}
     >
-      <AppBar.Item className={styles.logoContainer} href="/" tag="a">
-        <Logo className={styles.logo} />
+      <AppBar.Item className={styles.logoContainer}>
```

Comment:
We have `AppBar.Brand` which got forgotten and I don't even remember what it does.

Reply:
The brand should get deprecated or rewritten. It doesn't align correctly IMHO.